app/views/users/show.html.haml

```diff
@@ -0,0 +1,36 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest('knwang@gmail.com'.downcase)}?s=40")
```

Comment:
I would recommend moving the gravatar url to some helper method. Also you could use `= image_tag` for better readability.